Source/Mosa.Utility.Configuration/Name.cs

```diff
@@ -5,6 +5,8 @@ namespace Mosa.Utility.Configuration;
 public static class Name
 {
 	public const string AppLocation_Bochs = "AppLocation.Bochs";
+	public const string AppLocation_Bochs_BIOS = "AppLocation.Bochs.BIOS";
```

Comment:
Update documentation

Reply:
I was going to do this in a separate PR, is it fine if I do so?